map/framework.cpp

```diff
@@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
 
 void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
```

Comment:
Но здесь-то код не очевидный, если не начать грепать цепочки вызовов.

> On 26.3.2016, at 05.51, ygorshenin <notifications@github.com> wrote:
> 
> In map/framework.cpp:
> 
> > @@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
> >  
> >  void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
> Код лучшая документация? :) Твои слова вроде как )
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>